tests/algorithms/test_cutmix.py

```diff
@@ -81,7 +81,7 @@ def test_cutmix_algorithm(self, fake_data, alpha, uniform_sampling, minimal_stat
 
         algorithm = CutMix(alpha=alpha, uniform_sampling=uniform_sampling, interpolate_loss=interpolate_loss)
         state = minimal_state
-        state.model = ComposerClassifier(torch.nn.Flatten())
+        state.model = ComposerClassifier(torch.nn.Flatten(), num_classes=y_fake.shape[0])
```

Comment:
Is this right? above, `y_fake = torch.randint(num_classes, size=(N,))`, so I think `y_fake.shape[0]` is the size of the dataset.